evalml/tests/conftest.py

```diff
@@ -830,6 +830,22 @@ def X_y_regression():
     return X, y
 
 
+@pytest.fixture
+def X_y_multiseries_regression():
```

Comment:
I think I'm going to utilize this for my VARMAX testing. In that case, does it make sense to have it align more with the inputs we expect for a mutliseries regressor (e.g. a `series_id` column and the column names having `series_id` value suffixes)

Reply:
Good callout that this should more closely match the actual expected input - I wrote these before I wrote the stacking and unstacking functions and never updated it 😅 - same with why these tests don't use `multiseries_ts_data_unstacked`. I'll refactor the test fixtures a bit to be consolidated and up to date.